change_analyzer/sequences_diff.py

```diff
@@ -194,64 +234,91 @@ def update_verdicts_and_comments(self, valid: bool):
 
     def validate_steps_output(self, series: pd.Series):
         """Validate if each step's output is as expected"""
+        step_index = int(series['StepId'].split()[1])
+        step_action = series[f'{SequenceRecorder.COL_ACTION_TO_PERFORM}_actual']
 
+        print(f"Step {step_index} action: {step_action}")
         image_expected = series[f'{SequenceRecorder.COL_ACTION_IMAGE_AFTER}_expected']
         image_actual = series[f'{SequenceRecorder.COL_ACTION_IMAGE_AFTER}_actual']
 
         # Validate actual vs expected images are the same
         valid = self.validate_actual_vs_expected_images(image_actual, image_expected)
 
-        # Save images
-        expected_image_filename = f"expected_screenshot_step{len(self.verdicts) + 2}_{self.report_date}.png"
-        actual_image_filename = f"actual_screenshot_step{len(self.verdicts) + 2}_{self.report_date}.png"
-        self.save_step_images_from_arrays(image_actual, image_expected, actual_image_filename, expected_image_filename)
-
         # Update verdicts and comments based on the step validation
         self.update_verdicts_and_comments(valid)
 
-    def add_initial_step_to_sequences_data_components(self):
-        """Add an initial step, Open Application, validate actual and expected images and save them"""
+    @staticmethod
+    def _get_outline_color(diff_type):
+        """Define the outline color for an element, based on the diff type"""
+        if diff_type == 'UpdateAttrib':
+            return 'orange'
+        if diff_type == 'DeleteAttrib':
+            return 'red'
+        if diff_type == 'InsertAttrib':
+            return 'green'

```

Comment:
The method could be replaced with dictionary `{"UpdateAttrib": "orange", "DeleteAttrib"....}`